### ISSUE TITLE
Copying image to RAM will not work if image not fits 10% ram

### ISSUE THREAD

**Issue by kvaps** [ltsp/ltsp#102]
Hi @alkisg 
I just found one problem that initramfs mounting `/run` with option `size=10%`:

```bash
# /usr/share/initramfs-tools/init
mount -t tmpfs -o "noexec,nosuid,size=10%,mode=0755" tmpfs /run
```
But it might be not enough to fit whole image.

Any way image will be hidden in base system because another tmpfs will be mounted on top of `/run/initramfs/ltsp` by `overlay()` function

Possible solution is to update this option before copying image:
```
mount -o remount,size=100% /run
```
And save image outside of `/run/initramfs/ltsp` directory, eg into `/run/initramfs/`

Or move https://github.com/ltsp/ltsp/blob/151a14e6c9ad95a2e95a5f181dcfccd793b591e7/ltsp/client/initrd-bottom/55-initrd-bottom.sh#L59-L62 before https://github.com/ltsp/ltsp/blob/151a14e6c9ad95a2e95a5f181dcfccd793b591e7/ltsp/client/initrd-bottom/55-initrd-bottom.sh#L32-L40

but in this way boot with IMAGE_TO_RAM optiob will be failed because we need to run
https://github.com/ltsp/ltsp/blob/151a14e6c9ad95a2e95a5f181dcfccd793b591e7/ltsp/client/initrd-bottom/55-initrd-bottom.sh#L41

before 
https://github.com/ltsp/ltsp/blob/151a14e6c9ad95a2e95a5f181dcfccd793b591e7/ltsp/client/initrd-bottom/55-initrd-bottom.sh#L59-L62

Any possible ideas?

**Comment by kvaps** [comment]
I think we can move
https://github.com/ltsp/ltsp/blob/151a14e6c9ad95a2e95a5f181dcfccd793b591e7/ltsp/client/initrd-bottom/55-initrd-bottom.sh#L41
outside of `overlay()` function, or perform the check if `$tmpfs` already mounted, then skip mount

**Comment by alkisg** [comment]
Can't IMAGE_TO_RAM just mount a new, bigger tmpfs under /run/initramfs/ltsp?

**Comment by alkisg** [comment]
Fix committed in 147a6593d60a4aaa0105ab7b378f546ca01b9bc2.

@kvaps, or anyone else affected, could you please try the latest build from the [proposed PPA](https://ltsp.org/advanced/proposed-ppa/) and see if it solves your issue without any regressions, so that we upload LTSP 20.01 to Debian? Thanks!

**Comment by kvaps** [comment]
now it seem working fine, thanks
